test/e2e/v1beta1/scripts/gh-actions/run-e2e-tune-api.py

```diff
@@ -93,4 +162,4 @@ def objective(parameters):
         # Delete the Experiment.
         logging.info("---------------------------------------------------------------")
         logging.info("---------------------------------------------------------------")
-        katib_client.delete_experiment(exp_name, exp_namespace)
+        katib_client.delete_experiment(exp_name, exp_namespace)
```

Comment:
I think we need to delete the former experiment before we run another experiment. Otherwise, we may run into `xxx experiment alreay exists` error.